presto-main/src/main/java/com/facebook/presto/operator/scalar/StringFunctions.java

```diff
@@ -733,4 +733,16 @@ public static Slice toUtf8(@SqlType("varchar(x)") Slice slice)
     {
         return slice;
     }
+
+    @Description("returns the codepoint value of a single character string")
```

Comment:
Make this say "Unicode code point" to be consistent with the description for `chr()`